src/ASM/LR/ASMu2Dmxrecovery.C

```diff
@@ -59,11 +59,11 @@ bool ASMu2Dmx::assembleL2matrices (SparseMatrix& A, StdVector& B,
   if (continuous && !wg) return false;
 
   double dA = 0.0;
-  Vectors phi(m_basis.size());
-  Matrices dNdu(m_basis.size());
+  Vectors phi(2);
+  Matrices dNdu(2);
```

Comment:
Can also do `std::array<Vector,2> phi` and `std::array<Matrix,2> dNdu` I persume